patches/pathnames.h.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/pathnames.h b/pathnames.h
+index f7ca5a7..865d6a5 100644
+--- a/pathnames.h
++++ b/pathnames.h
+@@ -47,6 +47,48 @@
+ #define _PATH_SSH_PROGRAM		"/usr/bin/ssh"
+ #endif
+ 
++#ifdef __MVS__
++#include <libgen.h>
++#include <sys/ps.h>
++#include <unistd.h>
++#include <stdlib.h>
++#include <stdio.h>
++
++
++char* getRelativeSSHPath() {
```

Comment:
would it make sense to push this to zoslib? I know you brought this up before for getprogname - maybe we push it up with a new name (like you did) so that it doesn't conflict with getprogname ?

Reply:
should getRelativeSSHPath be static since it's in a header file?